src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -7990,9 +8001,17 @@ private static string WinInternalGetLinkType(string filePath)
                     REPARSE_DATA_BUFFER_SYMBOLICLINK reparseDataBuffer = Marshal.PtrToStructure<REPARSE_DATA_BUFFER_SYMBOLICLINK>(outBuffer);
 
                     if (reparseDataBuffer.ReparseTag == IO_REPARSE_TAG_SYMLINK)
+                    {
                         linkType = "SymbolicLink";
+                    }
                     else if (reparseDataBuffer.ReparseTag == IO_REPARSE_TAG_MOUNT_POINT)
+                    {
                         linkType = "Junction";
+                    }
+                    else if (reparseDataBuffer.ReparseTag == IO_REPARSE_TAG_APPEXECLINK)
+                    {
+                        linkType = "AppExeCLink";
+                    }
```

Comment:
It seems time to use `switch` expression or statement.

Reply:
Good point, I had changed it in the other location, but not here.  Will Fix.